readinglist/sessions/__init__.py

```diff
@@ -0,0 +1,24 @@
+class SessionStorageBase(object):
```

Comment:
I believe there are a bunch of libs which are doing that. Among these ones, we have one from Ben, which is named baker, and is used for pyramid. https://pypi.python.org/pypi/pyramid_beaker.

Relevant?